frontend/src/components/footer.js

```diff
@@ -181,12 +173,12 @@ const Footer = () => {
               <h2 className="text-Shades-0 text-sm md:text-lg font-normal mb-0 ml-1 tracking-medium-wide md:tracking-normal">
                 2023
               </h2>
-              <a
+              <Link
                 href="https://annarbor.hmcc.net"
```

Comment:
should this redirect to current gatsby site?

Reply:
im not sure, i think it probably should. ill update it to point to the new site